collection.php

```diff
@@ -1,11 +1,28 @@
 <?php
-$db = new PDO('mysql:host=db;dbname=project2collector', 'root', 'password');
-$db->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
-$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
 
-$query = $db->prepare("SELECT `name`, `definition`, `synonyms`, `example_sentence` FROM `collection`");
-$query->execute();
 
+ class Word {
```

Comment:
rename the file 'Word.php' as this will help with unit testing in the next task

and all the links/require_once uses for it aswell